composer.json

```diff
@@ -19,7 +19,9 @@
   "require": {
     "php": ">=7.1",
     "symfony/console": "^3.2",
-    "vlucas/phpdotenv": "^2.4"
+    "vlucas/phpdotenv": "^2.4",
```

Comment:
Just recently read [a thread on symfony's dotenv](https://github.com/symfony/dotenv) vs vlucas's. Was this considered?